util/kb.c

```diff
@@ -1314,21 +1314,19 @@ redis_add_nvt (kb_t kb, const nvti_t *nvt, const char *filename)
   if (rep != NULL)
     freeReplyObject (rep);
 
-  element = nvt->prefs;
-  if (g_slist_length (element))
+  if (nvti_pref_len (nvt))
     rep = redis_cmd (kbr, "DEL oid:%s:prefs", nvti_oid (nvt));
-  while (element)
+  for (i = 0; i < nvti_pref_len (nvt); i ++)
```

Comment:
There should not be a space before the ++.

Reply:
Fixed.